### PR TITLE
fix(core): allow all origins for profile and verification api

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

Allow access from all origins for UserRoute, which includes `/profile` and `/verifications`.

Because thoese APIs are for end users directly.

## Testing

Local tested and integration tests.

## Checklist

- [ ] `.changeset`
- [ ] unit tests
- [x] integration tests
- [ ] necessary TSDoc comments

### CODE REVIEW

File: packages/core/src/routes/init.ts [hunk 1]
```diff
@@ -53,11 +54,15 @@ import wellKnownRoutes from './well-known/index.js';
...
 const createRouters = (tenant: TenantContext) => {
+  const { adminUrlSet, cloudUrlSet } = EnvSet.values;
+
   const interactionRouter: AnonymousRouter = new Router();
+  interactionRouter.use(koaCors(adminUrlSet, cloudUrlSet));
   /** @deprecated */
   interactionRoutes(interactionRouter, tenant);
 
   const experienceRouter: AnonymousRouter = new Router();
+  interactionRouter.use(koaCors(adminUrlSet, cloudUrlSet));
```
Comment (simeng-li) [inline packages/core/src/routes/init.ts:65]: experienceRouter ？

File: packages/core/src/routes/init.ts [hunk 2]
```diff
@@ -102,8 +107,15 @@ const createRouters = (tenant: TenantContext) => {
   }
 
   const anonymousRouter: AnonymousRouter = new Router();
+  interactionRouter.use(koaCors(adminUrlSet, cloudUrlSet));
```
Comment (simeng-li) [inline packages/core/src/routes/init.ts:110]: anonymousRouter？

File: packages/core/src/routes/init.ts [hunk 3]
```diff
@@ -135,9 +135,8 @@ const createRouters = (tenant: TenantContext) => {
 
 export default function initApis(tenant: TenantContext): Koa {
   const apisApp = new Koa();
-
   const { adminUrlSet, cloudUrlSet } = EnvSet.values;
-  apisApp.use(koaCors(adminUrlSet, cloudUrlSet));
+  apisApp.use(koaCors([adminUrlSet, cloudUrlSet], ['/profile', '/verifications']));
```
Comment (simeng-li) [inline packages/core/src/routes/init.ts:139]: nit: better use a share const for these account API prefixes

**Review by simeng-li — APPROVED** [review]
overall LGTM

File: packages/core/src/routes/init.ts (hunk 3 above)
Comment (wangsijie) [reply packages/core/src/routes/init.ts:139]: good idea, I'll do it in the PR of renaming `/profile` to `/account`

**Pull request merged**